src/lib.rs

```diff
@@ -51,21 +52,26 @@ unsafe fn read_ptr_mut(slc: &[u8]) -> *mut u8 {
 }
 
 pub fn read_u32(slc: &[u8]) -> u32 {
-	(slc[0] as u32) + (slc[1] as u32).shl(8) + (slc[2] as u32).shl(16) + (slc[3] as u32).shl(24)
+	LittleEndian::read_u32(slc)
```

Comment:
wrappers still necessary? Or we could just use `LittleEndian::read_u32(slc)` and so on?

Reply:
oh you will then need to explicitly import byteorder crate

and it also hides endianness behind pwasm-std abstraction (so we may change how integers/pointers/etc. are interpreted at some point)